packages/rich-text-editor/src/EditableRichTextContent/EditableRichTextContent.module.scss

```diff
@@ -19,10 +19,17 @@
   border-color: $color-blue-500;
 }
 
-// Temporary solution for focus styling the content area
-// (the problem with this is that it shows this styling when focusing a link within the content
-.editableContainer:focus-within :global(.ProseMirror) {
-  background-color: $color-gray-200;
+.editableContainer .hiddenButton {
```

Comment:
this duplicated the VisuallyHidden component styles

Reply:
VisuallyHidden doesn't have `classNameOverride` yet but you could add it and then you could still use it?

Reply:
that's true, would save on the duplication of the hidden styles!

Reply:
Do you think it should be in a seperate PR or just bundle this one in?

Reply:
You can bundle if you want, but it does mean you're tied to a single release type (patch or minor) for both things

Reply:
hmmm does feel like I should break it up into `feat` for the VisuallyHidden and `refactor` for this